nodejs/awsx/apigateway/api.ts

```diff
@@ -606,17 +717,23 @@ function addStaticRouteToSwaggerSpec(
     // Create a bucket to place all the static data under.
     bucket = bucket || new aws.s3.Bucket(safeS3BucketName(name), undefined, parentOpts);
 
+    // Lazily initialize auth names
```

Comment:
i'm not sure what this comment means.  doesn't seem to add much value if it's just restating that we create auth-names if route.authorizers is present.

Reply:
I had seen comments somewhere else where we called out lazily initializing a variable, so I was following that. Probably not needed in this case